mule-deployer/src/main/java/org/mule/tools/deployment/cloudhub2/RequestBuilderCh2.java

```diff
@@ -88,19 +88,16 @@ protected RuntimeFabricDeploymentSettings resolveDeploymentSettings(RuntimeFabri
   }
 
   public Object createConfiguration() {
-    Map<String, Object> properties = null;
```

Comment:
it seems that here you're undoing your latest change https://github.com/mulesoft/mule-maven-plugin/commit/3d4fc0b3f53fa5f686238bb9944597de567d3224

Reply:
thanks.. updated and tested